xml/art_sle_ha_pmremote.xml

```diff
@@ -144,37 +158,50 @@
        A virtual machine that runs the &pmremote; daemon.
        A guest node is configured using the <option>remote-node</option>
        metadata option of a resource agent such as
-       <systemitem>ocf:pacemaker:VirtualDomain</systemitem>.
+       <systemitem>ocf:pacemaker:VirtualDomain</systemitem> (see use case&nbsp;2
+       in <xref linkend="tab.ha.pmremote.usecases"/>)
       </para>
      </listitem>
     </varlistentry>
    </variablelist>
-   <table xml:id="tab.ha.pmremote.comparison">
-    <title>Comparison of Regular HA Stack with and without &pmremote;</title>
+
+   <table xml:id="tab.ha.pmremote.usecases">
+    <title>Common Use Cases of &pmremote;</title>
     <tgroup cols="2">
+     <colspec colwidth="1*"/>
+     <colspec colwidth="1*"/>
      <thead>
       <row>
-       <entry>Regular Cluster Stack</entry>
-       <entry>Extended Cluster Stack with &pmremote;</entry>
+       <entry align="center">Use Case 1</entry>
+       <entry align="center">Use Case 2</entry>
+      </row>
+      <row>
+       <entry>Cluster Stack with Physical &pmremote; Node</entry>
+       <entry>Cluster Stack with Virtual &pmremote; Guest Nodes</entry>
       </row>
      </thead>
      <tbody>
       <row>
        <entry><informalfigure>
         <mediaobject>
          <imageobject>
-          <imagedata fileref="ha_pmremote-without.svg" width="90%"/>
+          <imagedata fileref="ha_pmremote-phys-remote-nodes.svg" width="90%"/>
          </imageobject>
         </mediaobject>
-       </informalfigure></entry>
+       </informalfigure>
+       <remark>toms 2017-07-11: Is the graphic above correct?</remark></entry>
```

Comment:
For the node "charlie" in use case 1, it should be one "Pacemaker Remote" instead of virtual machines running on "Physical host".